pkg/controller/sparkapplication/sparkui_test.go

```diff
@@ -18,6 +18,8 @@ package sparkapplication
 
 import (
 	"fmt"
+	extensions "k8s.io/api/extensions/v1beta1"
```

Comment:
These imports should go under the golang built-in ones.

Reply:
done